test/test.cpp

```diff
@@ -328,7 +328,10 @@ int main()
 
     int x, y;
     auto image = stbi_load( "image.jpg", &x, &y, nullptr, 4 );
-
+    if(image == nullptr)
+    {
+        std::cerr << "Could not find image (are you running this executable in the test/ folder ?), skipping" << std::endl;
```

Comment:
This seems like a misconfiguration on the CMake part. Can you make the image file be copied to the build / install dir instead?